packages/shared/src/test/ecsql-snippets/ECSqlJoinSnippets.test.ts

```diff
@@ -5,21 +5,23 @@
 
 import { expect } from "chai";
 import { ResolvablePromise } from "presentation-test-utilities";
-import { ECClass, ECNavigationProperty, ECRelationshipClass, IMetadataProvider } from "../../../hierarchies/ECMetadata";
-import { createRelationshipPathJoinClause } from "../../../hierarchies/queries/ecsql-snippets/ECSqlJoinSnippets";
-import { trimWhitespace } from "../../../hierarchies/Utils";
-import { ClassStubs, createClassStubs } from "../../Utils";
+import * as sinon from "sinon";
+import { createRelationshipPathJoinClause } from "../../shared/ecsql-snippets/ECSqlJoinSnippets";
+import * as m from "../../shared/Metadata";
+import { trimWhitespace } from "../../shared/Utils";
+import { createMetadataProviderStub } from "../MetadataProviderStub";
 
+import EC = m.EC;
```

Comment:
Why is this needed? Why not `import { EC } from "../../shared/Metadata";`?

Reply:
Was used as a workaround for something that is now irrelevant. Cleaned up.